app/controllers/application_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class ApplicationController < ActionController::Base
+  include SessionsHelper
+  include AttendingsHelper
```

Comment:
Really ought not have to include Helper methods in your Controller because it tells me there is application logic rather than view specific logic in your helpers.
Helper methods are directly accessible to views and do not have to be included in the application controller to be accessed by views, however, if you there are methods that are controller specific in your helpers they really ought to be defined in application controller directly.